aea/cli/fingerprint.py

```diff
@@ -48,10 +41,10 @@
 
 
 @click.group()
-@pass_ctx
-def fingerprint(ctx: Context):
+@click.pass_context
+@check_aea_project
```

Comment:
Ok we cannot actually run it here, otherwise you won't be able to fingerprint a resource.

Basically, the flow would be:
1. scaffold or develop your package
2. try to run aea > it points out that the fingerprint of these packages is no longer correct
3. you update the fingerprint for the packages you developed with these commands.
4. you successfully run.

Reply:
true... I'll fix that.

Reply:
that should be documented... I'll open an issue.